demisto_sdk/commands/format/update_pre_process_rules.py

```diff
@@ -35,18 +35,11 @@ def format_file(self) -> Tuple[int, int]:
     def run_format(self) -> int:
         try:
             click.secho(f'\n======= Updating file: {self.source_file} =======', fg='white')
-            self.set_version_to_default()
-            self.remove_unnecessary_keys()
```

Comment:
intentional?

Reply:
Already done this method on update_json.
Duplication.